benchmark/sirun/run-one-variant.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const { exec, stdio } = require('./run-util')
+
+process.env.DD_INSTRUMENTATION_TELEMETRY_ENABLED = 'false'
+
+const env = Object.assign({}, process.env, { DD_TRACE_STARTUP_LOGS: 'false' })
```

Comment:
Is there any change by not assigning `DD_TRACE_STARTUP_LOGS` to process.env directly? This code will work and is fine as such, I was just surprised about mixing the way envs are handled here.